<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QDialog" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>481</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>等待连接...........</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>年龄：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ageSpinBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>276</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sexComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>男</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>女</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>身高：</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="highDoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
